frontend/taipy-gui/src/components/Taipy/DateRange.tsx

```diff
@@ -137,6 +139,32 @@ const DateRange = (props: DateRangeProps) => {
     const handleChangeStart = useCallback((v: Date | null) => handleChange(v, true), [handleChange]);
     const handleChangeEnd = useCallback((v: Date | null) => handleChange(v, false), [handleChange]);
 
+    useEffect(() => {
+        if (props.defaultDates) {
```

Comment:
`defaultDates` is part of the dynamic property `dates`

You need to define another property for handling disabled dates